### DISCUSSION TITLE
On-Air can't add device, or change organizations name

### DISCUSSION THREAD

**Discussion by bloodnovski** [zauberzeug/nicegui#2996] [Q&A]
### Question

Hi, i sponsored the on-air program via github, however, i cant add another device to my orgnization, nor change the organizations name, made a typo when first creating it.

Can this be somehow resolved ?
Thanks in advance,

Miodrag.

**Comment by rodja** [comment]
Thanks for sponsoring 👍
Please send your current organization name and the corrected name to nicegui@zauberzeug.com. We will shortly update the database to include your extended permissions.
